lib/getNewThresholds.js

```diff
@@ -15,10 +16,11 @@ const getNewThresholds = (
     // Only update threshold if new coverage is higher than
     // current threshold + margin
     if (nextCoverage > threshold + margin) {
+      const next = Math.min(nextCoverage, maxThreshold ?? 100)
```

Comment:
Can't we skip updating if `nextCoverage` is greater than `maxThreshold` instead?

```suggestion
    if (nextCoverage > threshold + margin && nextCoverage <= maxThreshold) {
```

Reply:
not quite. if the nextCoverage is bigger than the targetThreshold, then we still want to update it to the targetThreshold.

But if the previous threshold was already the target threshold, then we don't need to update it. I added that condition.